quarkus/admin/src/main/java/org/apache/polaris/admintool/BootstrapCommand.java

```diff
@@ -38,20 +39,37 @@ public class BootstrapCommand extends BaseCommand {
   List<String> realms;
 
   @CommandLine.Option(
-      names = {"-c", "--credential"},
-      paramLabel = "<realm,clientId,clientSecret>",
+      names = {"-c", "--credentials"},
+      description = "Principal credentials to bootstrap. Must be a valid JSON array e.g. " +
+          "[{\"realm\": \"my-realm\", \"principal\": \"root\", \"clientId\": \"polaris\", \"clientSecret\": \"p4ssw0rd\"}]")
+  String credentials;
+
+  @CommandLine.Option(
+      names = {"-p", "--print-credentials"},
       description =
-          "Root principal credentials to bootstrap. Must be of the form 'realm,clientId,clientSecret'.")
-  List<String> credentials;
+          "Print root credentials to stdout")
+  boolean printCredentials;
 
   @Override
   public Integer call() {
     warnOnInMemory();
 
+    if (credentials == null || credentials.isEmpty()) {
+      if (!printCredentials) {
+        throw new IllegalArgumentException("Specify either `--credentials` or `--print-credentials` to ensure" +
```

Comment:
It's a simple error message. Stack trace is not valuable. `spec.commandLine().getErr().println(...)` might be more user-friendly.